app/src/main/java/a3/com/convo/fragments/ProfileDetailsFragment.java

```diff
@@ -47,5 +55,36 @@ public void onClick(View view) {
             }
         });
 
+        ParseUser user = ParseUser.getCurrentUser();
+        if (user == null) {
+            Log.e("ProfileDetailsFragment", "Somehow user was logged out of parse?");
+            return;
+        }
+        String user_name = user.getString(Constants.NAME);
```

Comment:
lets do camelCase here.. all variables in general would be camel case.. username in this case.. I think you renamed like this since its a conflict with the text view above.. you can name it as usernameView for textview and username for this variable.